lms/djangoapps/grades/signals/handlers.py

```diff
@@ -144,6 +145,8 @@ def score_published_handler(sender, block, user, raw_earned, raw_possible, only_
             modified=score_modified_time,
             score_db_table=ScoreDatabaseTableEnum.courseware_student_module,
         )
+        if isinstance(block, ScorableXBlockMixin):
```

Comment:
I agree with the principle, but I wanted to make this explicit because we've already encountered some places where we run into naming collisions with existing blocks, so at least until we're happy that our design is relatively final, *and* we've been able to audit existing blocks for method name collisions, I want to explicitly call out ScorableXBlock subclasses.  I'll put a comment here to refactor to use ducktyping once that has happened.